requirements-unit.in

```diff
@@ -19,4 +19,7 @@ ops
 pytest
 pytest-mock
 pytest-lazy-fixture
--r requirements.in
+serialized-data-interface
```

Comment:
Could you please add a comment here as well that this is required because of the 2259 issue? This way we don't forget to remove it once it is solved.

Reply:
Done!
https://github.com/canonical/kubeflow-profiles-operator/pull/229/commits/dc61cd305d7e20100cc07dfc410d25e454892cf0